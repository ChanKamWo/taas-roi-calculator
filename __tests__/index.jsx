import React from 'react';
import renderer from 'react-test-renderer';
import App from '../src/App';

test('Render App', () => {
  const component = renderer.create(
    <App />,
  );

  let app = component.toJSON();
  expect(app).toMatchSnapshot();
});