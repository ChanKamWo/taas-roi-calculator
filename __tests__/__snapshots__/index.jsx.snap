// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render App 1`] = `
Array [
  <header
    className="components-Header-styles___header___3H5Ffz"
  >
    <div
      className="components-Header-styles___content-center___2t5GcU"
    >
      <img
        alt="logo"
        className="components-Header-styles___logo___2tbxu8"
        src=""
      />
      <h1
        className="components-Header-styles___display-2___2-sJ2U"
      >
        TAAS ROI CALCULATOR
      </h1>
    </div>
  </header>,
  <div
    className="containers-Landing-styles___page___1kL9Uv"
    id="landing"
  >
    <div
      className="containers-Landing-styles___row___1bb8K3"
    >
      <div
        className="containers-Landing-styles___col___jhce-L"
      >
        <div
          className="sticky-outer-wrapper"
          style={Object {}}
        >
          <div
            className="sticky-inner-wrapper "
            style={
              Object {
                "position": "relative",
                "top": "",
                "transform": "translate3d(0,0px,0)",
                "zIndex": undefined,
              }
            }
          >
            <div
              className="containers-Landing-styles___left-section___1Gh0WH"
            >
              <h3
                className="upper-case containers-Landing-styles___heading-3___2bu8zb containers-Landing-styles___text-gradient___1GZA4v"
              >
                Hello. Welcome to the Talent as a Service ROI calculator
              </h3>
              <p
                className="containers-Landing-styles___text___v10vWP"
              >
                With this handy tool you can easily compare hiring a direct FTE vs using our innovative staffing solution. Simply input some details below to see your customized report and how to rapidly increase your team’s output and productivity.
              </p>
            </div>
          </div>
        </div>
      </div>
      <div
        className="containers-Landing-styles___col___jhce-L"
      >
        <form
          autoComplete="off"
          className="components-CalculatorForm-styles___calculator-form___2M9TfG"
          noValidate={true}
        >
          <div
            className="components-CalculatorForm-styles___dropdown___1t5llT"
          >
            <div
              className="dropdownContainer components-Dropdown-styles___container___BqT3EC components-Dropdown-styles___lgSize___TuSF8Z"
              onBlurCapture={[Function]}
              onFocusCapture={[Function]}
            >
              <div
                className="components-Dropdown-styles___relative___1hoItl"
              >
                <div
                  className="Select is-searchable Select--single"
                >
                  <div
                    className="Select-control"
                    onKeyDown={[Function]}
                    onMouseDown={[Function]}
                    onTouchEnd={[Function]}
                    onTouchMove={[Function]}
                    onTouchStart={[Function]}
                  >
                    <div
                      className="Select-multi-value-wrapper"
                      id="react-select-2--value"
                    >
                      <div
                        className="Select-placeholder"
                      >
                        Type of talent required
                      </div>
                      <div
                        className="Select-input"
                        style={
                          Object {
                            "display": "inline-block",
                          }
                        }
                      >
                        <input
                          aria-activedescendant="react-select-2--value"
                          aria-expanded="false"
                          aria-haspopup="false"
                          aria-owns=""
                          onBlur={[Function]}
                          onChange={[Function]}
                          onFocus={[Function]}
                          required={false}
                          role="combobox"
                          value=""
                        />
                      </div>
                    </div>
                    <span
                      className="Select-arrow-zone"
                      onMouseDown={[Function]}
                    >
                      <span
                        className="Select-arrow"
                        onMouseDown={[Function]}
                      />
                    </span>
                  </div>
                </div>
                <img
                  alt="dropdown-arraow-icon"
                  className="components-Dropdown-styles___iconDropdown___Viw0L9"
                  height="9"
                  src=""
                  width="15"
                />
              </div>
            </div>
          </div>
          <div
            className="components-CalculatorForm-styles___dropdown___1t5llT"
          >
            <div
              className="dropdownContainer components-Dropdown-styles___container___BqT3EC components-Dropdown-styles___lgSize___TuSF8Z"
              onBlurCapture={[Function]}
              onFocusCapture={[Function]}
            >
              <div
                className="components-Dropdown-styles___relative___1hoItl"
              >
                <div
                  className="Select is-searchable Select--single"
                >
                  <div
                    className="Select-control"
                    onKeyDown={[Function]}
                    onMouseDown={[Function]}
                    onTouchEnd={[Function]}
                    onTouchMove={[Function]}
                    onTouchStart={[Function]}
                  >
                    <div
                      className="Select-multi-value-wrapper"
                      id="react-select-3--value"
                    >
                      <div
                        className="Select-placeholder"
                      >
                        Number of people required
                      </div>
                      <div
                        className="Select-input"
                        style={
                          Object {
                            "display": "inline-block",
                          }
                        }
                      >
                        <input
                          aria-activedescendant="react-select-3--value"
                          aria-expanded="false"
                          aria-haspopup="false"
                          aria-owns=""
                          onBlur={[Function]}
                          onChange={[Function]}
                          onFocus={[Function]}
                          required={false}
                          role="combobox"
                          value=""
                        />
                      </div>
                    </div>
                    <span
                      className="Select-arrow-zone"
                      onMouseDown={[Function]}
                    >
                      <span
                        className="Select-arrow"
                        onMouseDown={[Function]}
                      />
                    </span>
                  </div>
                </div>
                <img
                  alt="dropdown-arraow-icon"
                  className="components-Dropdown-styles___iconDropdown___Viw0L9"
                  height="9"
                  src=""
                  width="15"
                />
              </div>
            </div>
          </div>
          <div
            className="components-CalculatorForm-styles___dropdown___1t5llT"
          >
            <div
              className="dropdownContainer components-Dropdown-styles___container___BqT3EC components-Dropdown-styles___lgSize___TuSF8Z"
              onBlurCapture={[Function]}
              onFocusCapture={[Function]}
            >
              <div
                className="components-Dropdown-styles___relative___1hoItl"
              >
                <div
                  className="Select is-searchable Select--single"
                >
                  <div
                    className="Select-control"
                    onKeyDown={[Function]}
                    onMouseDown={[Function]}
                    onTouchEnd={[Function]}
                    onTouchMove={[Function]}
                    onTouchStart={[Function]}
                  >
                    <div
                      className="Select-multi-value-wrapper"
                      id="react-select-4--value"
                    >
                      <div
                        className="Select-placeholder"
                      >
                        Industry
                      </div>
                      <div
                        className="Select-input"
                        style={
                          Object {
                            "display": "inline-block",
                          }
                        }
                      >
                        <input
                          aria-activedescendant="react-select-4--value"
                          aria-expanded="false"
                          aria-haspopup="false"
                          aria-owns=""
                          onBlur={[Function]}
                          onChange={[Function]}
                          onFocus={[Function]}
                          required={false}
                          role="combobox"
                          value=""
                        />
                      </div>
                    </div>
                    <span
                      className="Select-arrow-zone"
                      onMouseDown={[Function]}
                    >
                      <span
                        className="Select-arrow"
                        onMouseDown={[Function]}
                      />
                    </span>
                  </div>
                </div>
                <img
                  alt="dropdown-arraow-icon"
                  className="components-Dropdown-styles___iconDropdown___Viw0L9"
                  height="9"
                  src=""
                  width="15"
                />
              </div>
            </div>
          </div>
          <div
            className="components-CalculatorForm-styles___dropdown___1t5llT"
          >
            <div
              className="dropdownContainer components-Dropdown-styles___container___BqT3EC components-Dropdown-styles___lgSize___TuSF8Z"
              onBlurCapture={[Function]}
              onFocusCapture={[Function]}
            >
              <div
                className="components-Dropdown-styles___relative___1hoItl"
              >
                <div
                  className="Select is-searchable Select--single"
                >
                  <div
                    className="Select-control"
                    onKeyDown={[Function]}
                    onMouseDown={[Function]}
                    onTouchEnd={[Function]}
                    onTouchMove={[Function]}
                    onTouchStart={[Function]}
                  >
                    <div
                      className="Select-multi-value-wrapper"
                      id="react-select-5--value"
                    >
                      <div
                        className="Select-placeholder"
                      >
                        Company size
                      </div>
                      <div
                        className="Select-input"
                        style={
                          Object {
                            "display": "inline-block",
                          }
                        }
                      >
                        <input
                          aria-activedescendant="react-select-5--value"
                          aria-expanded="false"
                          aria-haspopup="false"
                          aria-owns=""
                          onBlur={[Function]}
                          onChange={[Function]}
                          onFocus={[Function]}
                          required={false}
                          role="combobox"
                          value=""
                        />
                      </div>
                    </div>
                    <span
                      className="Select-arrow-zone"
                      onMouseDown={[Function]}
                    >
                      <span
                        className="Select-arrow"
                        onMouseDown={[Function]}
                      />
                    </span>
                  </div>
                </div>
                <img
                  alt="dropdown-arraow-icon"
                  className="components-Dropdown-styles___iconDropdown___Viw0L9"
                  height="9"
                  src=""
                  width="15"
                />
              </div>
            </div>
          </div>
          <div
            className="components-CalculatorForm-styles___radio-group___3Z-r3k"
          >
            <span
              className="components-CalculatorForm-styles___label___xAZSlw"
            >
              Cost of Living
               
              <span>
                <svg
                  className="components-CalculatorForm-styles___help-icon___eRyfkI"
                  height="16"
                  width="16"
                  xmlns="http://www.w3.org/2000/svg"
                  xmlnsXlink="http://www.w3.org/1999/xlink"
                >
                  <defs>
                    <path
                      d="M8 0c4.4 0 8 3.6 8 8s-3.6 8-8 8-8-3.6-8-8 3.6-8 8-8zm0 2C4.7 2 2 4.7 2 8s2.7 6 6 6 6-2.7 6-6-2.7-6-6-6zm1 5v5H7V7h2zM8 4a1 1 0 1 1 0 2 1 1 0 0 1 0-2z"
                      id="a"
                    />
                  </defs>
                  <use
                    fill="#AAA"
                    fillRule="evenodd"
                    xlinkHref="#a"
                  />
                </svg>
              </span>
              Cost of Living Increases
            </span>
            <div
              className="radioButtonContainer components-RadioButton-styles___radioButtonContainer___zyNQ_D components-RadioButton-styles___lgSize___7HVGko"
            >
              <div
                className="radioButton components-RadioButton-styles___radioButton___1nF-Nc"
              >
                <label
                  className="components-RadioButton-styles___container___2iMOc0"
                >
                  <input
                    checked={false}
                    onChange={[Function]}
                    type="radio"
                  />
                  <span
                    className="components-RadioButton-styles___checkmark___19bpDm"
                  />
                  <span
                    className="components-RadioButton-styles___label___1sGbVN"
                  >
                    Low
                  </span>
                </label>
              </div>
              <div
                className="radioButton components-RadioButton-styles___radioButton___1nF-Nc"
              >
                <label
                  className="components-RadioButton-styles___container___2iMOc0"
                >
                  <input
                    checked={false}
                    onChange={[Function]}
                    type="radio"
                  />
                  <span
                    className="components-RadioButton-styles___checkmark___19bpDm"
                  />
                  <span
                    className="components-RadioButton-styles___label___1sGbVN"
                  >
                    Medium
                  </span>
                </label>
              </div>
              <div
                className="radioButton components-RadioButton-styles___radioButton___1nF-Nc"
              >
                <label
                  className="components-RadioButton-styles___container___2iMOc0"
                >
                  <input
                    checked={false}
                    onChange={[Function]}
                    type="radio"
                  />
                  <span
                    className="components-RadioButton-styles___checkmark___19bpDm"
                  />
                  <span
                    className="components-RadioButton-styles___label___1sGbVN"
                  >
                    High
                  </span>
                </label>
              </div>
            </div>
          </div>
          <hr
            className="components-CalculatorForm-styles___horizontal-rule___2J-lhZ"
          />
          <div
            className="components-CalculatorForm-styles___input___3Fj5gg"
          >
            <div
              className="textInputContainer components-TextInput-styles___container___1EE3aK components-TextInput-styles___lgSize___1sT8v6"
            >
              <input
                className=""
                defaultValue=""
                onBlur={[Function]}
                onChange={[Function]}
                onKeyPress={[Function]}
                placeholder="First name"
                readOnly={false}
                type="text"
              />
            </div>
          </div>
          <div
            className="components-CalculatorForm-styles___input___3Fj5gg"
          >
            <div
              className="textInputContainer components-TextInput-styles___container___1EE3aK components-TextInput-styles___lgSize___1sT8v6"
            >
              <input
                className=""
                defaultValue=""
                onBlur={[Function]}
                onChange={[Function]}
                onKeyPress={[Function]}
                placeholder="Last name"
                readOnly={false}
                type="text"
              />
            </div>
          </div>
          <div
            className="components-CalculatorForm-styles___input___3Fj5gg"
          >
            <div
              className="textInputContainer components-TextInput-styles___container___1EE3aK components-TextInput-styles___lgSize___1sT8v6"
            >
              <input
                className=""
                defaultValue=""
                onBlur={[Function]}
                onChange={[Function]}
                onKeyPress={[Function]}
                placeholder="Company"
                readOnly={false}
                type="text"
              />
            </div>
          </div>
          <div
            className="components-CalculatorForm-styles___input___3Fj5gg"
          >
            <div
              className="textInputContainer components-TextInput-styles___container___1EE3aK components-TextInput-styles___lgSize___1sT8v6"
            >
              <input
                className=""
                defaultValue=""
                onBlur={[Function]}
                onChange={[Function]}
                onKeyPress={[Function]}
                placeholder="Work email"
                readOnly={false}
                type="text"
              />
            </div>
          </div>
          <div
            className="components-CalculatorForm-styles___footer___3bCHd_"
          >
            <button
              className="src-shared-components-Button-style___button___ngAZY2 src-shared-components-buttons-themes-primary___button___1vJGbc src-shared-components-Button-style___regular___3HACs4 src-shared-components-buttons-themes-primary___regular___i6lrwz"
              onClick={[Function]}
              onMouseDown={null}
              style={Object {}}
              type="button"
            >
              SEE YOUR RESULTS
            </button>
          </div>
        </form>
      </div>
    </div>
  </div>,
  <footer
    className="components-Footer-styles___footer___3AMYgu"
  >
    <p
      className="components-Footer-styles___content-center___3CWEyh"
    >
      By clicking “schedule a taas call” you agree to our Terms and Privacy Policy. We are never going to sell your data or send you spam. Your information is being used for communication purposes only. Wage and salalry data:
       
      <a
        href="https://www.glassdoor.com/"
      >
        glassdoor.com
      </a>
      . Bonus, equity, other benefits & employee overhead data: MIT study
       
      <a
        href="https://web.mit.edu/e-club/hadzima/pdf/how-much-does-an-employee-cost.pdf"
      >
        How Much Does An Employee Cost?
      </a>
      . Recruiting & time to hire data: workable.com
       
      <a
        href="https://resources.workable.com/tutorial/faq-recruitment-budget-metrics"
      >
        Recruiting Costs: Budget and Cost per Hire | Recruiting Metrics FAQ
      </a>
    </p>
  </footer>,
]
`;
